supporting_content/metrics/pricingquantity.md

```diff
@@ -0,0 +1,96 @@
+# Column: PricingQuantity
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | Cost and Usage Report        | lineItem/UsageAmount (NOTE: how to handle reservation/TotalReservedUnits|
+| GCP       | BigQuery Billing Export      | usage.amount_in_pricing_units (NOTE: usage.amount provides UsageQuantity) |
+| Microsoft | Cost Details                 | Not available (NOTE: While Quantity is the closest, it provides UsageQuantity and not PricingQuantity) |
```

Comment:
Interesting. I thought Azure quantity was closer to PricingQuantity than to UsageQuantity. I would be interested by any link or documentation.

Indeed, in case of reservations, Azure documents to calculate on-demand equivalent price by multiplying UnitPrice by Quantity. Reference: https://learn.microsoft.com/en-us/azure/cost-management-billing/reservations/understand-reserved-instance-usage-ea

Reply:
I thought so too, but Michael pointed this out, and he is the authority on Azure :) The FOCUS Converters WG will be reviewing the suggested provider mappings in detail.